src/twisted/conch/test/test_manhole.py

```diff
@@ -244,10 +242,21 @@ def finished(ign):
                     + defaultFunctionName.encode("utf-8"),
                     b"Exception: foo bar baz",
                     b">>> done",
-                ]
+                ],
             )
 
-        return done.addCallback(finished)
+        done.addCallback(finished)
+        return done
+
+    def test_ExceptionWithCustomExcepthook(
+        self,
+    ):
+        """
+        Raised exceptions are handled the same way even if L{sys.excepthook}
```

Comment:
The same way as .... ?

I understand the purpose of the test in the context of the PR.
But by reading the test code and test docstring, it is not obvious what is tested here.

It also might be just me being tired after work.